content/en/tracing/troubleshooting/tracer_debug_logs.md

```diff
@@ -30,9 +30,36 @@ To enable debug mode for the Datadog Java Tracer, set the flag `-Ddd.trace.debug
 
 {{< programming-lang lang="python" >}}
 
-To enable debug mode for the Datadog Python Tracer, set the environment variables `DD_TRACE_DEBUG=true` and `DD_CALL_BASIC_CONFIG=true` when using `ddtrace-run`.
+The steps for enabling debug mode Datadog Python Tracer, depends on which version of the tracer your application is using. Choose the scenario that applies:
+
+### Scenario 1: dd-trace-py version 1.3.0 or higher
+
+1. Set the `DD_TRACE_DEBUG` environment variable to `true` to enable debug mode.
+
+2. To route debug logs to a log file, set `DD_TRACE_LOG_FILE` with a filename that tracer logs should be written to, for example, `DD_TRACE_LOG_FILE=ddtrace_logs.log`. 
```

Comment:
Not sure if it's clear where on the filesystem `ddtrace_logs.log` will be? Is this relative to the current directory at the time the app is started?

Reply:
>Is this relative to the current directory at the time the app is started
By default, yes. This example will go to `<currentdirectory>/ddtrace_logs.log`.

To reference a different file, someone could do `/var/logs/ddtrace_logs.log` or similar. Should I add an example to route it elsewhere too to be more clear?

Reply:
```suggestion
2. To route debug logs to a log file, set `DD_TRACE_LOG_FILE` with a filename that tracer logs should be written to, relative to the current working directory. For example, `DD_TRACE_LOG_FILE=ddtrace_logs.log`. 
```

maybe something like this?